src/enums.py

```diff
@@ -211,3 +211,25 @@ class Direction(StrEnum):
 class EntityState(StrEnum):
     IDLE = "idle"
     WALK = "walk"
+
+
+class LAYER(IntEnum):
+    WATER = auto()
```

Comment:
I'd set this first value to 0 so you actually can infer the next values in the enum in the code, especially since it's an IntEnum